server/express.js

```diff
@@ -1,13 +1,33 @@
 const bodyParser = require('body-parser');
 const express = require('express');
 const path = require('path');
+const morgan = require('morgan');
+const connect = require('./utils/db');
+const config = require('./../config/config');
+// const personRouter = require('./resources/person/person.router');
+// const certificationRouter = require('./resources/certification/certification.router');
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, '../public')));
 
 app.use(bodyParser.json());
+app.use(morgan('dev'));
```

Comment:
Let's move this file above the express.static line to also log requests for static files.